vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -840,7 +860,7 @@ public void success(PinCodeInfo pinCodeInfo) {
                 VimeoClient.sContinuePinCodeAuthorizationRefreshCycle = true;
                 mPinCodeAuthorizationTimer.scheduleAtFixedRate(
                         new PinCodePollingTimerTask(pinCodeInfo, mPinCodeAuthorizationTimer,
-                                                    pinCodeInfo.getExpiresIn(), authCallback, sSharedInstance,
+                                                    pinCodeInfo.getExpiresIn(), authCallback, getInstance(),
```

Comment:
You're right. I changed it to this because the parameter is annotated `@NotNull` and it was giving a lint warning that the instance could have been null. I assumed that there wouldn't be a consequence because it would crash when it was null. Looking into the code deeper though, it looks like the client is immediately placed in a `WeakReference`, so there was no harm passing null to the not-null annotated parameter. So, I'm going to revert back to using the field and change the annotation on the parameter.